src/index.js

```diff
@@ -7,6 +7,7 @@
  * [![Build Status](https://travis-ci.org/nxus/pipeliner.svg?branch=master)](https://travis-ci.org/nxus/pipeliner)
  * 
  * A framework for creating and running data pipelines.  Data pipelines have stages, which are made of an arbitrary number of tasks.  Stages and tasks are run in serial: once a task completes, the next task in the pipeline is executed.  
+ * A task may complete synchronously, or asynchronously through use of a promise. 
```

Comment:
I guess this is true, but the expectation is that whether it is an a async or sync return, all the tasks are processed serially.  What are you trying to clarify here?